.github/workflows/runtests.yml

```diff
@@ -148,7 +148,7 @@ jobs:
       - run: python -m pip install --upgrade pip
       - run: pip install mpmath gmpy2 matplotlib numpy scipy aesara ipython \
                          symengine cython llvmlite wurlitzer \
-                         autowrap numexpr 'antlr4-python3-runtime==4.7.*' \ 
+                         autowrap numexpr 'antlr4-python3-runtime==4.10' \
```

Comment:
Should this also have `.*`?

Reply:
The release is just 4.10 not 4.10.0

Reply:
But what happens when 4.10.1 is released? Shouldn't it work as well?

Reply:
4.10.1 has been released, can keep the wildcard.